docs/src/main/asciidoc/logging.adoc

```diff
@@ -118,6 +118,19 @@ In this example, the logger name would be `com.example.MyService`.
 WARNING: Only use the `Log` API in application classes, not in external dependencies.
 `Log` method calls that are not processed by Quarkus at build time will throw an exception.
 
+[[log-api-extension-warning]]
+==== Important Note on Using `io.quarkus.logging.Log` in Extensions
+
+While the `Log` API simplifies logging in application classes, it should not be used in extension modules or external dependencies. The following considerations apply:
+
+* `io.quarkus.logging.Log` depends on Quarkus bytecode transformations that occur at build time.
+* In extension modules, the use of `Log` may work if the module has a Jandex index. However, this behavior is not officially supported and might lead to unreliable logging.
+
+For extension development:
+
+* Use standard loggers like `Logger.getLogger(Class)` instead of `io.quarkus.logging.Log`.
```

Comment:
```suggestion
* Use standard loggers like `org.jboss.logging.Logger.getLogger(String)` instead of `io.quarkus.logging.Log`.
```

I (still) do not recommend using the class name as the category name of a logger. In retrospect, I wish I had made it use the package name instead when that constructor was used...

Reply:
Thank you for the suggestion! I've made the change locally as you recommended.